src/clj/rems/api/services/workflow.clj

```diff
@@ -15,20 +15,28 @@
 (def ^:private validate-workflow-body
   (s/validator workflow/WorkflowBody))
 
+(defn invalid-forms-error [forms]
+  (let [invalid (seq (filter (comp nil? form/get-form-template :form/id) forms))]
+    (when invalid
+      {:success false
+       :errors [{:type :invalid-form
+                 :forms invalid}]})))
+
 (defn create-workflow! [{:keys [user-id organization type title handlers forms]}]
   (util/check-allowed-organization! organization)
```

Comment:
I wonder if we should put this inside the `or` though it's more of the assert type. Perhaps instead of `check-allowed-organization! it should be called `assert-allowed-organization!` or something.

Reply:
It's outside the `or` in lots of places like `rems.api.services.form`. I think the `check-` name was chosen to be consistent with `check-user` and `check-roles`. I think `assert-` would sound too severe.